openhub/data.py

```diff
@@ -51,5 +51,7 @@ def get_data(for_what):
             except Exception as ex:
                 logger.error(ex)
                 break
+        if type(data) is not list:
```

Comment:
This is a bug which is being fixed in https://github.com/coala/community/pull/152

I will remove it from here when that is merged.